### DISCUSSION TITLE
Trilium interview

### DISCUSSION THREAD

**Discussion by zadam** [TriliumNext/Trilium#4157] [Show and tell]
I gave an interview on Trilium, it might be of interest also here: https://console.substack.com/p/console-169

**Comment by XXXJetfireXXX** [comment]
It's really nice to see you getting more attention. Keep up the good work!
Also, good thinking on avoiding burnouts. That is indeed an important matter, especially when the project is being maintained by a few people or even one person. That is why, as a user, I prefer not to try and hasten the developer just by telling him to do things faster: creativity can't be coerced. If I wanted to speed up the process and had the skills needed, I would contribute. x)

**Comment by biffbaxter** [comment]
Good interview! I think the burnout and support piece is important to mention as you did. I have supported opensource projects for 20+ years and its a real thing. Life and income generating work definitely take priority, and sometimes its hard for users to understand that. Keep on working at whatever pace works for you, and enjoy interesting challenges. :) - Thanks for sharing.
